lib/Sema/TypeCheckGeneric.cpp

```diff
@@ -244,15 +244,15 @@ void TypeChecker::checkGenericParamList(GenericSignatureBuilder *builder,
 
   // Determine where and how to perform name lookup for the generic
   // parameter lists and where clause.
-  TypeResolutionOptions options;
+  TypeResolutionOptions options = None;
```

Comment:
Instead of None, why not have a 'default' context?

Reply:
I wanted to avoid error-of-omission bugs.